code/modules/admin/holder2.dm

```diff
@@ -52,7 +55,24 @@ var/list/admin_datums = list()
 		owner.deadmin_holder = null
 		owner.add_admin_verbs()
 
-
+/proc/GenerateToken()
```

Comment:
what are these tokens for?

Reply:
It is for HrefToken, but I think we don't need HrefToken at now moment.